### PR TITLE
add goal reached before conclude sweep

### PR DESCRIPTION
**Pull request by kangkelvin**

### CODE REVIEW

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
+{
+  "configurations": [
+    {
+      "browse": {
+        "databaseFilename": "",
+        "limitSymbolsToIncludedHeaders": true
+      },
+      "includePath": [
+        "/home/kelvin/aison_ws/autoware.ai.gpu/install/wf_simulator/include/**",
+        "/home/kelvin/aison_ws/autoware.ai.gpu/install/trafficlight_recognizer/include/**",
```
Comment (bkolligs) [inline .vscode/c_cpp_properties.json]: can you add the .vscode to the .gitignore

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        "/home/kelvin/aison_ws/autoware.ai/install/autoware_system_msgs/lib/python2.7/dist-packages",
+        "/home/kelvin/aison_ws/autoware.ai/install/autoware_camera_lidar_calibrator/lib/python2.7/dist-packages",
+        "/home/kelvin/aison_ws/autoware.ai/install/autoware_msgs/lib/python2.7/dist-packages",
+        "/home/kelvin/aison_ws/autoware.ai/install/autoware_map_msgs/lib/python2.7/dist-packages",
+        "/home/kelvin/aison_ws/autoware.ai/install/autoware_launcher/lib/python2.7/dist-packages",
+        "/home/kelvin/aison_ws/autoware.ai/install/autoware_lanelet2_msgs/lib/python2.7/dist-packages",
+        "/home/kelvin/aison_ws/autoware.ai/install/autoware_config_msgs/lib/python2.7/dist-packages",
+        "/home/kelvin/aison_ws/autoware.ai/install/autoware_can_msgs/lib/python2.7/dist-packages",
+        "/home/kelvin/dragoon_ws/devel/lib/python2.7/dist-packages",
+        "/opt/ros/melodic/lib/python2.7/dist-packages"
```
Comment (bkolligs) [inline .vscode/settings.json]: same as above

File: explore/src/explore.cpp [hunk 3]
```diff
@@ -372,10 +372,14 @@ void Explore::sendLastSweepAndStop()
 {
   move_base_client_.cancelAllGoals();
   exploring_timer_.stop();
+  // Send goal reached to go to sweep
   dragoon_messages::stateCmd stateMsg;
-  stateMsg.event = "CONCLUDE SWEEP";
+  stateMsg.event = "GOAL REACHED";
```
Comment (bkolligs) [inline explore/src/explore.cpp:377]: looks good

**Review by bkolligs — CHANGES_REQUESTED** [review]

**Comment by kangkelvin** [comment]
ugh vscode

**Pull request merged**